include/motorsProcessor.h

```diff
@@ -53,7 +53,14 @@ class Motor {
     }
 
     int getRPM () {
-        return currentRPM;
+        // Read the current encoder count
```

Comment:
The encoder counter must be read directly inside Motor.getRPM(). We should respond to the client with the actual number of encoder ticks since the last read. Please reset the counter immediately after reading it. Do not cache this value.